### ISSUE TITLE
dockerbuilder pod spawned with debug

### ISSUE THREAD

**Issue by bacongobbler** [deis/builder#289]
This was mentioned by @gabrtv today in the standup that building a docker image on v2-beta1 flips on debug. I have not confirmed but this should be turned off

**Comment by gabrtv** [comment]
Thanks @bacongobbler.  I would also suggest taking a pass at cleaning up the output generally, and making it as presentable as possible in preparation for 2.0.0.

**Comment by mboersma** [comment]
This issue unfortunately won't be fixed in a release until v2.0.0-beta3. My apologies to those who demo Deis Workflow. ping @gabrtv.

The current workaround is to change these variables in `deis-builder-rc.yaml`:
```
DEBUG=false
DOCKERBUILDER_IMAGE_NAME=quay.io/deisci/dockerbulder:git-8715439
```
That will pull in a dockerbuilder image just post-beta2 that includes deis/dockerbuilder#46 and provides cleaner output during `git push deis master`.
